doc/nrf/gs_programming.rst

```diff
@@ -7,15 +7,20 @@ Building and programming an application
    :local:
    :depth: 2
 
-The recommended way of building and programming an |NCS| application is to use the Nordic Edition of the |SES| (SES) IDE.
-
 |application_sample_definition|
 
-.. note::
+These user guides contain platform-specific instructions for building and programming.
```

Comment:
What are "these" user guides?

Reply:
It looks like there was a line missing from the original content. This is added back in. It's referring to the hardware guidelines for the boards.